### ISSUE TITLE
Issue while building the docker image

### ISSUE THREAD

**Issue by janismdhanbad** [zacwellmer/WorldModels#6]
Hello,

Thanks for the repo. I am facing an error while building the docker:

```
Step 23/46 : RUN python3 -m pip install --no-cache-dir ${TF_PACKAGE}${TF_PACKAGE_VERSION:+==${TF_PACKAGE_VERSION}}
 ---> Running in ebae39a5f72c
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.5/dist-packages/pip/__main__.py", line 21, in <module>
    from pip._internal.cli.main import main as _main
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/cli/main.py", line 60
    sys.stderr.write(f"ERROR: {exc}")
                                   ^
SyntaxError: invalid syntax

```
Could you please suggest a fix for this?

Thanks

**Comment by zacwellmer** [comment]
I think the issue has to do with pip and python3.5. I'm away for the weekend but can submit a fix when I'm back. If you fix it, feel free to submit a PR

**Comment by OperaRuss** [comment]
Was this ever resolved?  Am facing this same issue.

**Comment by nicksawhney** [comment]
Also running into this issue at the moment

**Comment by zacwellmer** [comment]
addressed by #11
